### ISSUE TITLE
http module field conflict

### ISSUE THREAD

**Issue by suwubee** [labring/FastGPT#703]
**例行检查**

[//]: # '方框内填 x 表示打钩'

- [x] 我已确认目前没有类似 issue
- [x] 我已完整查看过项目 README，以及[项目文档](https://doc.fastgpt.in/docs/intro/)
- [x] 我使用了自己的 key，并确认我的 key 是可正常使用的
- [x] 我理解并愿意跟进此 issue，协助测试和提供反馈
- [x] 我理解并认可上述内容，并理解项目维护者精力有限，**不遵循规则的 issue 可能会被无视或直接关闭**

**你的版本**

- [x] 公有云版本
- [ ] 私有部署版本

**问题描述**
HTTP模块中，字段不能为系统提交的字段，例如url，否则整个data不会生成或者数据丢失。
建议屏蔽或者提醒。
另外，在http输出结果中，只能输出最外层的数组，能否加一个功能来筛选内层数组，看了下其他组件都没有这个功能，因为只是数组提取，也用不到AI来执行。

**复现步骤**
HTTP字段设置为url

**预期结果**
屏蔽系统字段的添加或者提醒，增加http输出多维数组

**相关截图**

**Comment by c121914yu** [comment]
> Bot detected the issue body's language is not English, translate it automatically. 👯👭🏻🧑‍🤝‍🧑👫🧑🏿‍🤝‍🧑🏻👩🏾‍🤝‍👨🏿👬🏿         
----    
**Routine inspection**

[//]: # 'Fill x in the box to indicate tick'

- [x] I have confirmed that there is currently no similar issue
- [x] I have fully reviewed the project README, as well as [project documentation](https://doc.fastgpt.in/docs/intro/)
- [x] I used my own key and confirmed that my key can be used normally
- [x] I understand and am willing to follow up on this issue, assist in testing and provide feedback
- [x] I understand and acknowledge the above content, and understand that project maintainers have limited energy. **Issues that do not follow the rules may be ignored or closed directly**

**your version**

- [x] Public cloud version
- [ ] private deployment version

**Problem Description**
In the HTTP module, fields cannot be fields submitted by the system, such as URL, otherwise the entire data will not be generated or the data will be lost.
It is recommended to block or remind.
In addition, in the http output result, only the outermost array can be output. Is it possible to add a function to filter the inner array? I checked that other components do not have this function, because it is only array extraction and does not use AI. implement.

**Steps to reproduce**
HTTP field set to url

**expected outcome**
Block the addition or reminder of system fields, and increase the http output multi-dimensional array

**Related screenshots**
